### DISCUSSION TITLE
[Projects beta] Allow sorting by more than one column

### DISCUSSION THREAD

**Discussion by codytodonnell** [community/community#8895] [Projects and Issues] [answered]
It would be nice to be able to sort issues by at least two different columns simultaneously. For example, if you have both a "priority" column and an "effort" column, it would be very helpful to be able to sort by both so that highest priority and lowest effort issues appear at the top.

**Comment by rmcnews** [comment]
You can do `Group by values` on the first column and `sort` the second column to get a similar effect.

**Comment by rileybroughten** [comment, marked as answer]
You can now sort by a secondary field on your project views! Check out the [changelog](https://github.blog/changelog/2023-07-13-github-issues-projects-july-13th-update/#file_cabinet-improvements-to-sorting-fields) for more details ✨

**Comment by shamminjiti** [comment]
Why stop at 2 fields? Can't you allow sorting based on as many fields as possible?
